.github/workflows/changelog.yaml

```diff
@@ -30,4 +30,4 @@ jobs:
           title: 'docs: updated CHANGELOG.md'
           commit-message: 'docs: updated CHANGELOG.md'
           branch: create-pull-request/changelog
-          signoff: true
```

Comment:
Same here

Reply:
And other places in this PR

Reply:
there's no change here, it's just a newline at EOF, which GitHub and many editors point out. Mine automatically adds it when missing in any file